tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py27, py36
 
 [testenv]
 deps =
-    -rrequirements.txt
+    setuptools==44.1.1
```

Comment:
This applies to py27 alone right? If yes, maybe you can place it under `[testenv:py2]` instead